dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/runner/GlobalTaskDispatchWaitingQueue.java

```diff
@@ -29,7 +27,7 @@
 @Component
 public class GlobalTaskDispatchWaitingQueue {
 
-    private final PriorityBlockingQueue<DefaultTaskExecuteRunnable> queue = new PriorityBlockingQueue<>();
+    private final DelayQueue<DefaultTaskExecuteRunnable> queue = new DelayQueue<>();
```

Comment:
Should be DelayPriorityQueue?

Reply:
We cannot use PriorityQueue in TaskExecutor, since the task dispatch to master/worker then it will loss priority, we control the priority at Master dispatching